### ISSUE TITLE
Prior participation eligibility: must meet criteria for all studies in each list

### ISSUE THREAD

**Issue by becky-gilbert** [lookit/lookit-api#1306]
# TL;DR

If there are multiple studies in the 'must have participated' / 'must not have participated' lists for study eligibility, the child must meet the criteria for ALL of studies in the list.

# Narrative

As a researcher, if I add multiple studies to the 'must have participated' study criteria, I expect that children will only be eligible if they have participated in all (not just some) of the studies in the list. Similarly, if I add multiple studies to the 'must not have participated' list, I expect that children will only be eligible if they have not participated in any of the studies in the list.

# Acceptance Criteria

- [x] Participation eligibility checks will treat multiple studies with AND (rather than OR) logic, i.e. must have participated in all studies in the required list, must not have participated in any studies in the disallowed list
- [x] Tests
- [x] Make sure this is clear in documentation: https://github.com/lookit/lookit-docs/pull/365
